lib/logger.js

```diff
@@ -11,8 +11,10 @@ const fs = require('./util/unwrapped-core').fs
 // create bootstrapping logger
 module.exports = new Logger({
```

Comment:
now that we aren't redefining the module.exports you could just create the instance of logger 1 line above then on line 34 just reference `logger.configure`.  Either way this was a great idea and really streamlines this logger